exampleCourse/questions/demo/autograder/c/factorial/info.json

```diff
@@ -0,0 +1,14 @@
+{
+  "uuid": "c3b02ed9-3a2e-462a-9d18-f1b21cab65d1",
+  "title": "Recursive implementation of factorial",
+  "topic": "C",
+  "singleVariant": true,
+  "tags": ["jgfoster", "clang", "factorial", "c"],
+  "type": "v3",
+  "gradingMethod": "External",
+  "externalGradingOptions": {
+    "enabled": true,
+    "image": "prairielearn/grader-c",
+    "entrypoint": "/grade/tests/test.py"
```

Comment:
@jonatanschroeder the entrypoint can be omitted now, right?

Reply:
Indeed, removed in e3d8c32a9f2c1c0e559f07de321fa1e524329ef5.